pkg/testcase/cluster.go

```diff
@@ -49,37 +47,14 @@ func TestBuildCluster(cluster *shared.Cluster) {
 	}
 }
 
-// TestSonobuoyMixedOS runs sonobuoy tests for mixed os cluster (linux + windows) node.
-func TestSonobuoyMixedOS(deleteWorkload bool) {
-	sonobuoyVersion := customflag.ServiceFlag.External.SonobuoyVersion
-	err := shared.SonobuoyMixedOS("install", sonobuoyVersion)
+// TestDisplayClusterDetails used to display cluster details.
+func TestDisplayClusterDetails() {
```

Comment:
Are we not using this here? https://github.com/rancher/distros-test-framework/blob/main/pkg/template/helper.go#L79

Reply:
This method is still exposed publicly from the conformance file